### PR TITLE
Alert-in-body element created

### PR DESCRIPTION
**Pull request by yone920**
Alert-in-body element created.

### CODE REVIEW

File: src/elements/alert-in-body/alert-in-body.scss [hunk 1]
```diff
@@ -0,0 +1,11 @@
+.alert-in-body {
```
Comment (kimpenguin) [inline src/elements/alert-in-body/alert-in-body.scss:1]: @yone920 instead of `.alert-in-body`, could we rename the class to `.alert`?

**Review by kimpenguin — CHANGES_REQUESTED** [review]
@yone920 Looks good. Could you adjust the class name and also try to see if there is any styling adjustments you might want to make based on the `head_extras` in the twig files in the core? There's some template specific styling. Maybe the padding could be set with tailwind classes?

**Comment by yone920** [comment]
> @yone920 Looks good. Could you adjust the class name and also try to see if there is any styling adjustments you might want to make based on the `head_extras` in the twig files in the core? There's some template specific styling. Maybe the padding could be set with tailwind classes?

I have included the styling on the head_extras to the 'alert-in-body.scss. I am not if you refering the styles in the head-extra.

**Comment by kimpenguin** [comment]
@yone920 Could you also rename the slm files and element files to alert for consistency?

The head_extras are the .alert styling on some of the twig files in the core. It's wrapped within on some of the single.twig an archive.twig files`{% block head_extras %}{% endblock %}`. We would remove those in-template stylings to use this pattern.

**Review by kimpenguin — APPROVED** [review]

**Comment by yone920** [comment]
Merged to Main.

**Pull request closed without merging**